cypress/integration/search_for_property.spec.js

```diff
@@ -11,7 +11,7 @@ beforeEach(() => {
 describe('Search for property', () => {
   it('Search for property by postcode', () => {
     // Stub request for search on properties by postcode
-    cy.route('GET', 'api/v2/properties/?q=e9 6pt', '@propertiesList')
```

Comment:
Where did the v2 go ? 😀

Reply:
This is now mocking out the response from the Next JS API rather than the service API. But this has made me realise I'm missing a layer of tests - for the API routes (not just the util functions they call). I'll add those.

Reply:
Will add in a separate PR as discussed.